src/Microsoft.Graph.Core/Requests/AsyncMonitor.cs

```diff
@@ -42,61 +45,58 @@ public async Task<T> PollForOperationCompletionAsync(IProgress<AsyncOperationSta
         {
             while (!cancellationToken.IsCancellationRequested)
             {
-                using (var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, this.monitorUrl))
+                var requestInformation = new RequestInformation() { HttpMethod = Kiota.Abstractions.HttpMethod.GET , UrlTemplate = this.monitorUrl};
+                var nativeResponseHandler = new NativeResponseHandler();
+                await this.client.RequestAdapter.SendNoContentAsync(requestInformation, nativeResponseHandler).ConfigureAwait(false);
```

Comment:
same comment about native response handling vs the serialization layer provided by kiota

Reply:
I agree with you on this. I will action and resolve in an upcoming update.